src/helpers/storage.ts

```diff
@@ -0,0 +1,18 @@
+import type { IStoredKeys } from '../routes/connect/IStoredKeys';
+import StorageKeyNotFoundError from '../routes/connect/errors/StorageKeyNotFoundError';
+
+export function storageData(key: string, cryptoKey: string): void {
+    sessionStorage.setItem('cryptoKey', cryptoKey);
+    sessionStorage.setItem('privateKey', key);
+}
+
+export function getStorageData(): IStoredKeys {
```

Comment:
I can now see why you've named it in plural, but instead of calling it IStoredKeys call it IStoredKeyPair

Reply:
`function get(key: string): any`